web-local/src/lib/components/workspace/explore/utils.ts

```diff
@@ -1,5 +1,12 @@
-import type { MetricsViewDimension } from "@rilldata/web-common/runtime-client";
+import type {
+  MetricsViewDimension,
+  V1MetricsView,
+} from "@rilldata/web-common/runtime-client";
 
 export function getDisplayName(dimension: MetricsViewDimension) {
   return dimension?.label?.length ? dimension?.label : dimension?.name;
 }
+
+export function hasDefinedTimeSeries(metricsView: V1MetricsView) {
```

Comment:
This should be `useModelHasTimeSeries(...)` following the useSelector pattern described here: https://tkdodo.eu/blog/react-query-data-transformations#3-using-the-select-option

I think a good rule-of-thumb is to use that pattern wherever we call pure functions on query results.